content/docs/2.0/scalers/influxdb.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "InfluxDB"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on influxdb queries"
++++
+
+### Trigger Specification
+
+This specification describes the `influxdb` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      authToken: some-auth-token
+      organizationName: influx-org
+      authTokenFromEnv: INFLUXDB_AUTH_TOKEN
+      organizationNameFromEnv: INFLUXDB_ORG_NAME
+      serverURL: http://influxdb:8086
+      thresholdValue: '4'
+      query: |
+        from(bucket: "bucket-of-interest")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "stat")
+```
+
+**Parameter list:**
```

Comment:
Nit again but I would seperately list the `FromEnv` entries from the non-env ones and just mention that it does the same but read from an env variable.

For example:
```markdown
- `organizationName` is the organization name needed for the client to locate all [information](https://docs.influxdata.com/influxdb/v2.0/organizations/) contained in that organization such as buckets, tasks, etc.
- `organizationNameFromEnv` defines the organization name, similar to `organizationName`, but reads it from an environment variable on the scale target.
```